src/ResourceManager/Kusto/Commands.Kusto/Az.Kusto.psd1

```diff
@@ -118,7 +118,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = 'Initial version of the Kusto powershell interface'
+        ReleaseNotes = 'Initial version of the Kusto PowerShell interface'
```

Comment:
It would be nice to have the URI to Kusto documentation in the description field, but I won't hold it up for this.